src/t_stream.c

```diff
@@ -2205,9 +2205,9 @@ void xreadCommand(client *c) {
             streams_arg = i+1;
             streams_count = (c->argc-streams_arg);
             if ((streams_count % 2) != 0) {
-                addReplyError(c,"Unbalanced XREAD list of streams: "
-                                "for each stream key an ID or '$' must be "
-                                "specified.");
+                addReplyErrorFormat(c,"Unbalanced %s list of streams: "
```

Comment:
> ERR Unbalanced xreadgroup list of streams

since cmd->fullname is now lowercase, should we add a '' ?

```suggestion
                addReplyErrorFormat(c,"Unbalanced '%s' list of streams: "
```

Reply:
yes, come to think about it, we had a campaign recently to do something similar in many other places (use `'` and fullname)